prometheus/README.md

```diff
@@ -14,22 +16,21 @@ The Prometheus check is packaged with the Agent starting version 6.1.0.
 
 ### Configuration
 
-Edit the `prometheus.d/conf.yaml` file to add your different prometheus instances you want to retrieve metrics from.
+Edit the `prometheus.d/conf.yaml` file to to retrieve metrics from your prometheus instances.
```

Comment:
We do not collect metrics from prometheus instances.  We collect metrics from applications that expose OpenMetrics / Prometheus end points.

Reply:
Thanks for the feedback. PR #2445 added.